### PR TITLE
Fix for the filters bug on the results page

### PR DESCRIPTION
**Pull request by drewpotter**
Fixed the locale for the filters selection on the results page.

### CODE REVIEW

File: app/views/results/_filters.html.erb [hunk 1]
```diff
@@ -21,6 +21,7 @@
             <%= label_tag "filter[gender]", t('results.show_only_results_by_females') %>
           </p>
 
+          <%= hidden_field_tag "locale", I18n.locale %>
```
Comment (spencerldixon) [inline app/views/results/_filters.html.erb]: Should this not be sourcable from the controller? Might be a better way of doing it

Comment (drewpotter) [reply app/views/results/_filters.html.erb]: Ah sure, maybe I should create a locale variable in the controller and set it equal to this?

Comment (drewpotter) [reply app/views/results/_filters.html.erb]: Done!

File: app/controllers/results_controller.rb [hunk 2]
```diff
@@ -12,6 +12,7 @@ def index
   def show
     @results = AnswerSet.find_by(uuid: params[:uuid])
     @all_other_results = AnswerSet.last(1000).reject{ |r| r.id == @results.id }.pluck(:x_axis_scaled, :y_axis_scaled)
+    @locale = I18n.locale
```
Comment (spencerldixon) [inline app/controllers/results_controller.rb]: @drewpotter I meant the other way around, I dont think we need to pass this in at all from the front end, we should just be able to grab it in the filter logic in the controller

Comment (drewpotter) [reply app/controllers/results_controller.rb]: @spencerldixon I don't completely understand this. I think the params for the locale are not being sent when the filters are being selected. It's like it drops them off. This is why I added a hidden field to append those.

**Review by spencerldixon — APPROVED** [review]

**Pull request merged**